### PR TITLE
Rename PolymerElement.mixins to PolymerElement.behaviors

### PR DESCRIPTION
**Pull request by robliao**
`mixins` has been renamed to `behaviors` in https://github.com/Polymer/polymer/commit/940d975955a74ab4990c1f1651a8c9dc5caea388

### CODE REVIEW

File: polymer.externs.js [hunk 1]
```diff
@@ -53,11 +53,11 @@ PolymerElement.prototype.is;
 PolymerElement.prototype.extends;
 
 /**
- * An array of objects whose properties get mixed in to this element.
+ * An array of objects whose properties get added to this element.
```
Comment (jklein24) [inline polymer.externs.js:56]: Maybe just add an @see to https://www.polymer-project.org/1.0/docs/devguide/behaviors.html

**Comment by jklein24** [comment]
LGTM pending CLA

File: polymer.externs.js (hunk 1 above)
Comment (robliao) [inline polymer.externs.js:56]: Done

**Comment by robliao** [comment]
I'm now in the Google github org. See http://github.com/robliao.

**Comment by robliao** [comment]
I signed it!

**Pull request merged**